packages/widget_toolkit_biometrics/example/lib/main.dart

```diff
@@ -0,0 +1,191 @@
+import 'package:app_settings/app_settings.dart';
+import 'package:flutter/material.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:widget_toolkit/widget_toolkit.dart';
+import 'package:widget_toolkit_biometrics/widget_toolkit_biometrics.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Widget Toolkit Biometrics Demo',
+      theme: ThemeData.light().copyWith(
+        colorScheme: ColorScheme.fromSwatch(),
+        // You have to provide the [WidgetToolkitTheme]s in order for the
+        // [BiometricsSwitch] to work
+        extensions: [
+          WidgetToolkitTheme.light,
+        ],
+      ),
+      darkTheme: ThemeData.dark().copyWith(
+        colorScheme: ColorScheme.fromSwatch(),
+        extensions: [
+          // You have to provide the [WidgetToolkitTheme]s in order for the
+          // [BiometricsSwitch] to work
+          WidgetToolkitTheme.dark,
+        ],
+      ),
+      home: const MyHomePage(title: 'Widget Toolkit Biometrics Demo'),
+    );
+  }
+}
+
+class MyHomePage extends StatelessWidget {
+  const MyHomePage({super.key, required this.title});
+
+  final String title;
+
+  @override
+  Widget build(BuildContext context) => Scaffold(
```

Comment:
In order to make the build method more readable you can extract all closures as private methods